java-api-core/src/com/kinvey/java/store/UserStoreRequestManager.java

```diff
@@ -57,13 +58,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static com.kinvey.java.Constants.ACCESS_ERROR;
+
 /**
  * Created by Prots on 2/12/16.
  */
 public class UserStoreRequestManager<T extends BaseUser> {
 
 
-    public static final String USER_COLLECTION_NAME = "user";
+    public static final String USER_COLLECTION_NAME = "active_user_info";
```

Comment:
any special reason why you need to change the collection name for the local cache?

Reply:
I'm wondering if it would cause a breaking change

Reply:
`USER_COLLECTION_NAME` wasn't used in the SDK, but I agree that it's public constant, I'll create `ACTIVE_USER_COLLECTION_NAME` for `active_user_info` and return back `user`.

Reply:
I wouldn't like to use `user` for keeping user info, because this table name potentially can be busy.

Reply:
Updated.